.drone.star

```diff
@@ -224,6 +226,7 @@ def gui_tests(ctx, trigger = {}, depends_on = [], filterTags = [], version = "da
                              "CLIENT_REPO": "/drone/src/",
                              "MIDDLEWARE_URL": "http://testmiddleware:3000/",
                              "BACKEND_HOST": "http://owncloud/",
+                             "SECURE_BACKEND_HOST": "https://owncloud/",
```

Comment:
do we need to update this documentation for an additional environment? https://doc.owncloud.com/desktop/2.9/appendices/guitest.html#run-tests-using-docker

Reply:
I guess we should